datahub-web-react/src/app/search/SearchHeader.tsx

```diff
@@ -74,12 +75,16 @@ export const SearchHeader = ({
     entityRegistry,
 }: Props) => {
     const themeConfig = useTheme();
+    const appConfig = useAppConfig();
 
     return (
         <Header style={styles.header as any}>
             <LogoSearchContainer>
                 <Link to="/">
```

Comment:
One small thing - should we also update the login page?

Reply:
Will do in a followup- want this to get into the release :)